CMakeLists.txt

```diff
@@ -120,6 +120,28 @@ if ((ENABLE_HIP) AND (NOT ENABLE_KOKKOS))
   list(APPEND RAJA_PERFSUITE_DEPENDS blt::hip_runtime)
 endif()
 
+#
+# Are we using Caliper
+#
+set(ENABLE_CALIPER off CACHE BOOL "")
+set(RAJAPERF_USE_CALIPER off)
```

Comment:
Should this be a dependent option?

Reply:
The variable should be renamed to RAJA_PERF_USE_CALIPER and use that everywhere.

Reply:
We can make it a dependent option since cmake configure will abort here anyways 
if (ENABLE_CALIPER)
   find_package(caliper REQUIRED)
   
changing to RAJA_PERF_USE_CALIPER with add_definition (-DRAJA_PERF_USE_CALIPER) .. easy enough

Definitely appreciate the comments.

Reply:
We should prefix CMake variables with 'RAJA_PERFSUITE_' when they are unique to the PerfSuite and prefer to use standard ones provided by CMake or those provided by BLT when possible. In this case, ENABLE_CALIPER isn't provided by CMake or BLT, so it shouldn't be a Cmake dependent option.

Reply:
Ahh, gotcha